app/Http/Controllers/Api/V1/TeamController.php

```diff
@@ -83,6 +83,34 @@ public function index(Organization $suborganization)
         ], 200);
     }
 
+    /**
+     * Get All Organization Teams
+     *
+     * @urlParam suborganization required The Id of a suborganization Example: 1
+     *
+     * Get a list of the teams of an Organization.
+     *
+     * @responseFile responses/team/teams.json
+     *
+     * @return Response
+     */
+    public function getOrgTeams(Organization $suborganization)
+    {
+        $this->authorize('viewAny', [Team::class, $suborganization]);
+
+        $teams = $this->teamRepository->getOrgTeams($suborganization->id);
+
+        $teamDetails = [];
+        foreach ($teams as $team) {
+            $team = $this->teamRepository->getTeamDetail($team->id);
```

Comment:
I think we should combine getOrgTeams and getTeamDetail and take this code in repository.